analytics/config/config.go

```diff
@@ -17,6 +18,20 @@ func NewPBSAnalytics(analytics *config.Analytics) analytics.PBSAnalyticsModule {
 			glog.Fatalf("Could not initialize FileLogger for file %v :%v", analytics.File.Filename, err)
 		}
 	}
+	if analytics.Pubstack.Enabled {
```

Comment:
I've switched to an error log instead of a fatal. 
But the question is what do we want to do if we can't load a required module. Should we fail? If yes, I think we have to redesign this part to make it more testable.

Reply:
I think we should abort the startup if the analytics config cannot be processed.